lang/python/python3-drf-nested-routers/Makefile

```diff
@@ -0,0 +1,33 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=drf-nested-routers
+PKG_VERSION:=0.91
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+PKG_SOURCE:=drf-nested-routers-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/d/drf-nested-routers/
+PKG_HASH:=46e5c3abc15c782cafafd7d75028e8f9121bbc6228e3599bbb48a3daa4585034
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/drf-nested-routers-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-drf-nested-routers
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Nested resources for the Django Rest Framework
+  URL:=https://github.com/alanjds/drf-nested-routers
+  DEPENDS:=+python3-django-restframework
```

Comment:
As I see, yes (beside recursive dependencies on python3-django and python3 …)

Reply:
I would prefer to be explicit about:
* Matching [upstream's dependencies](https://github.com/alanjds/drf-nested-routers/blob/v0.91.0/setup.py#L78-L81), which would include `python3-django`
* Whether this package needs full `python3` or just `python3-light`

Reply:
I have difficulties to test which dependencies are needed for this package `drf-nested-routers` as python3-django depends on `python3`.

Using `django` with `python3-light`, i.e. if I remove all the other python3 packages, the runtests.py (without `pytest`) would need at least:
`python3-distutils python3-urllib python3-email python3-decimal python3-logging python3-cgi python3-codecs python3-pytz python3-unittest python3-xml python3-sqlite3`
But, the tests fail with `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`
I do not know if (some of) the following full `python3` packages are needed:
`python3-pydoc python3-cgitb python3-ncurses python3-ctypes python3-multiprocessing python3-gdbm python3-openssl python3-dbm python3-lzma python3-asyncio python3-pkg-resources`
The tests fail with the same error. So, maybe the only dependencies that are missing, are the ones for the tests ...

I think it is not worth to investigate that further before django is not using `python3-light` and selected packages instead of `python3`.

Reply:
Aside from a run test, a way to investigate which packages may be needed is to look for `import` lines in the code. For instance, if there are instances of `import xml`, then `python3-xml` is probably needed. If many of these packages are required, then requiring the full `python3` makes sense.

In this case, I didn't see any out-of-the-ordinary imports, so `python3-light` is probably sufficient. (But please double-check if you can.)

Reply:
Ah, now I understand what you did mean. I did remove the runtests directory and grep through the files for `import` lines: all of them (except the one with django and rest_framework) run in the python3 console after removing all python3 packages beside `python3-light`. If they do not use another way to import things that should be enough.